pkg/platform/provider/baremetal/conf/nvidia/nvidia-device.config

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
This file is a shell script, not a config. Consider renaming it.
Why put it into volume？Do we need change it？

Reply:
1. I am ok to correct the name, WDYT `enable-nvidia-device.sh`?
2. this script will not get changed,  put it into configmap so that `platform` component can mount it as volume to broadcast this script to gpu node for remote execute.

Reply:
1. `enable-nvidia-device.sh` LGTM
2. how about put it into provider-res image or tke-platform-controller image ?

Reply:
regarding to #2,  I prefer to keep current logic rather than include to provider-res. cm map offers great flexibility

Reply:
ok